docs/Compile.rst

```diff
@@ -485,11 +485,54 @@ other versions won't work against Dwarf Fortress due to ABI and STL incompatibil
 
 You can install Visual Studio 2015_ or 2017_ Community edition for free, which
 include all the features needed by DFHack. You can also download just the
-`build tools`_ if you aren't going to use Visual Studio to edit code.
+Visual C++ 2015 `build tools`_ if you aren't going to use Visual Studio to edit code.
+
+Option 1: Build Tools Only
+++++++++++++++++++++++++++
```

Comment:
To fix the doc build warnings, try an underline style that hasn't been used in the file yet, like backticks (```````````)